ilc/client.js

```diff
@@ -17,7 +17,7 @@ if (System === undefined) {
 }
 
 const registryConf = initSpaConfig();
-const router = new Router(registryConf);
+const router = new Router(registryConf, singleSpa.navigateToUrl);
```

Comment:
navigateToUrl is global var. why should we throw it inside router?

Reply:
This is not a global var. It is part of single spa. So, we need to test the router with DI first of all. If you have any solution on how to test it without DI we can discuss it.